pkg/api/router/router.go

```diff
@@ -78,8 +79,19 @@ func (r *Router) RegisterAPIKeyHandlers() {
 }
 
 func (r *Router) RegisterAnnotationsHandlers() {
-	h := api.NewAnnotationsHandler(r.Logger, r.AnnotationsService, httputils.NewDefaultHelper(r.Logger))
+	h := api.NewAnnotationsHandler(r.AnnotationsService, httputils.NewDefaultHelper(r.Logger))
 
 	x := r.PathPrefix("/annotations").Subrouter()
 	x.Methods(http.MethodPost).HandlerFunc(h.CreateAnnotation)
 }
+
+func (r *Router) RegisterAdhocHandlers() {
+	h := api.NewAdhocHandler(r.AdhocService, httputils.NewDefaultHelper(r.Logger))
+
+	x := r.PathPrefix("/adhoc/v1").Subrouter()
+	x.Methods(http.MethodGet).PathPrefix("/profiles").HandlerFunc(h.GetProfiles)
+	x.Methods(http.MethodGet).PathPrefix("/profile/{id:[0-9a-f]+}").HandlerFunc(h.GetProfile)
+	x.Methods(http.MethodGet).PathPrefix("/diff/{left:[0-9a-f]+}/{right:[0-9a-f]+}").HandlerFunc(h.GetProfileDiff)
+	x.Methods(http.MethodPost).PathPrefix("/upload").HandlerFunc(h.Upload)
+	x.Methods(http.MethodPost).PathPrefix("/upload-diff/").HandlerFunc(h.UploadDiff)
```

Comment:
I didn't find any actual usage of the endpoint in the frontend. Do we need it?

/cc @eh-am

Reply:
that is correct, i guess it's a left over